components/classes/summary_metrics.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/summarymetrics",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Summary Metrics",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["all"]
+  },
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/time-series"
+  ],
+  "description": "A Summary Metric record provides pre-summarized metrics with grouping dimensions.  An example would be the results of an SQL SELECT with a GROUP BY.",
+  "definitions": {
+    "summarymetrics": {
+      "properties": {
```

Comment:
This class doesn't add any additional properties on it's own.  It just provides a specific class on top of time-series that we can use to drive user-experience and data handling specifically for summaries.

Additional properties will be the user's use-case specific metrics and optional dimensions.